input/about/member-policy.md

```diff
@@ -0,0 +1,28 @@
+---
+title: .NET Foundation Membership Policy
+layout: _BasicPageLayout
+---
+
+ <!-- start -->
+<section class="page-section page-section">
```

Comment:
@BillWagner 
You can remove all html, it is superfluous here
https://jolly-grass-05beb181e-1053.westus2.azurestaticapps.net/about/member-policy

Reply:
@isaacrlevin Can you advise here? Adding this fixed the CSS, but it does seem to be duplicating the divs used for the bootstrap grid.

Reply:
The current version looks fine. Merging in